cloudtools/init_notebook.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/python
+import os
+import json
+from subprocess import check_output, call
+
+import sys
+if sys.version_info >= (3,0):
+    # Python 3 check_output returns a byte string
+    decode_f = lambda x: x.decode()
+else:
+    # In Python 2, bytes and str are the same
+    decode_f = lambda x: x
```

Comment:
This is duplicated below.  Want to put it in utils somewhere?

Reply:
I'm sort of uncomfortable doing that since init_notebook is deployed as a standalone init script, meant to be totally separate.

Reply:
Ah!  Good point.  Nevermind.